src/tensorlake/cli/deploy.py

```diff
@@ -138,63 +139,18 @@ def _deploy_applications(
         raise click.Abort
 
-    deployed_apps = filter_applications(functions)
-    try:
-        application = next(deployed_apps)
-        fn_config: _FunctionConfiguration = application.function_config
-
-        # Get parameter type from function signature
-        import inspect
-
-        sig = inspect.signature(application.original_function)
-        first_param = next(iter(sig.parameters.values()), None)
-
-        if first_param and first_param.annotation != inspect.Parameter.empty:
-            param_annotation = first_param.annotation
-
-            if hasattr(param_annotation, "model_json_schema") or hasattr(
-                param_annotation, "schema"
-            ):
-                schema = (
-                    getattr(param_annotation, "model_json_schema", lambda: None)()
-                    or getattr(param_annotation, "schema", lambda: {})()
-                )
-                properties = schema.get("properties", {})
-                field_examples = []
-
-                for field_name, field_schema in properties.items():
-                    # Handle different schema formats
-                    if "type" in field_schema:
-                        field_type_name = field_schema["type"]
-                    elif "anyOf" in field_schema:
-                        # Show all types in the union
-                        types = [
-                            item.get("type")
-                            for item in field_schema["anyOf"]
-                            if item.get("type")
-                        ]
-                        field_type_name = " | ".join(types) if types else "value"
-                    else:
-                        field_type_name = "value"
-                    field_examples.append(f'"{field_name}": <{field_type_name}>')
-
-                param_type = "{" + ", ".join(field_examples) + "}"
-            else:
-                type_name = getattr(param_annotation, "__name__", str(param_annotation))
-                param_type = f"<{type_name}>"
-        else:
-            param_type = "<value>"
-
-        click.echo(f"Deployed application: {fn_config.function_name}\n")
-        click.echo(
-            f"""To invoke the application, use the following curl command:
-curl -X POST {auth.base_url}/v1/namespaces/{auth.namespace}/applications/{fn_config.function_name} \\
+    for function in functions:
+        if function.application_config is not None:
+            func_name = function.function_config.function_name
+            click.echo(f"Deployed application: {func_name}\n")
+            click.echo(
+                f"""To invoke the application, use the following curl command:
+curl -X POST {auth.base_url}/v1/namespaces/{auth.namespace}/applications/{func_name} \\
   -H "Authorization: Bearer $TENSORLAKE_API_KEY" \\
   -H "accept: application/json" \\
   -H "Content-Type: application/json" \\
-  -d '{param_type}'
+  -d '{get_function_input_types(function)}'
 """,
-        )
-    except StopIteration:
```

Comment:
If this code path doesn't have any obvious bugs then it shouldn't fail, I agree that exception handling is not needed here for this reason. I'd leave a comment about this.